engine/resource/src/resource.cpp

```diff
@@ -460,7 +460,7 @@ void UpdateFactory(HFactory factory)
 {
     DM_PROFILE(__FUNCTION__);
     dmMessage::Dispatch(factory->m_Socket, &Dispatch, factory);
-    dmJobThread::Update(factory->m_JobThreadContext);
+    dmJobThread::Update(factory->m_JobThreadContext, 0);
```

Comment:
Should probably test loading times on HTML5 as well, as 0 means "1 job per frame", but if we have budget for it, we can process many more items per frame.